tmva/tmva/inc/TMVA/MethodRNN.h

```diff
@@ -105,7 +105,7 @@ class MethodRNN : public MethodBase
 
 protected:
 
-   void MakeClassSpecific( std::ostream&, const TString& ) const;
+   //void MakeClassSpecific( std::ostream&, const TString& ) const;
```

Comment:
Hi. Please amend the previous commit with these changes rather than creating a commit with message "Bugfixes" only.
Also, please refrain from adding commented out code. Thanks.